c10/test/util/bits16_test.py

```diff
@@ -0,0 +1,43 @@
+import torch
+from torch.testing._internal.common_utils import run_tests, TestCase
+from torch.utils._mode_utils import no_dispatch
+from torch.utils._pytree import tree_map
+
+class TensorSubclassDemo(torch.Tensor):
```

Comment:
I understand this is a demo class, but maybe can make its name more semantically meaningful? Like UInt16Tensor?

Reply:
I guess technically this is Int16Tensor